app/components/provider_interface/application_timeline_component.rb

```diff
@@ -7,7 +7,7 @@ def initialize(application_choice:)
       @application_choice = application_choice
     end
 
-    Event = Struct.new(:title, :actor, :date)
+    Event = Struct.new(:title, :actor, :date, :type, :id)
```

Comment:
Do we need `:id` any more?

Reply:
Adam is planning a route along the lines of `/notes/:id` for linking and possiblly editing or deleting a specific note. So, these fields will be needed for the links then. I can remove these for now.